main/py3-urlgrabber/APKBUILD

```
@@ -0,0 +1,28 @@
+# Contributor: Natanael Copa <ncopa@alpinelinux.org>
+# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
+pkgname=py3-urlgrabber
+pkgver=4.0.0
+pkgrel=0
+pkgdesc="A python module for fetching files"
+url="http://urlgrabber.baseurl.org/"
+arch="noarch"
+license="LGPL-2.0-or-later"
+depends="python3 py3-curl py3-six"
+makedepends="py3-setuptools"
+subpackages="$pkgname-doc"
+source="https://github.com/rpm-software-management/urlgrabber/releases/download/urlgrabber-${pkgver//./-}/urlgrabber-$pkgver.tar.gz"
+builddir="$srcdir"/urlgrabber-$pkgver
+
```

Would adding a `provides` and a `replaces` give the end user an upgrade path?

```suggestion
provides="py-urlgrabber=$pkgver-r$pkgrel" # for backward compatibility
replaces="py-urlgrabber" # for backward compatibility
```